sktime/networks/cntc.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+"""a combination of CCNN and CLSTM as feature extractors."""
+
+__author__ = ["James-Large, Withington, TonyBagnall", "AurumnPegasus"]
+
+from sktime.networks.base import BaseDeepNetwork
+from sktime.utils.validation._dependencies import _check_dl_dependencies
+
+_check_dl_dependencies(severity="warning")
+
+
+class CNNNetwork(BaseDeepNetwork):
```

Comment:
if you have base class for neural network classifiers, why are you not inheriting from it?
`BaseDeepClassifier`, not `BaseDeepNetwork`

Reply:
there is already a class named `CNNNetwork`

Reply:
ok, after discussion understood why this is inheriting from `BaseDeepNetwork` - the classifier would be somewhere else